rest-api-spec/src/main/resources/rest-api-spec/api/cat.segment_replication.json

```diff
@@ -0,0 +1,99 @@
+{
+  "cat.segment_replication":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/master/cat-segment_replication.html",
```

Comment:
We should not be including any further links to elasticsearch documentation anymore. This should be redirected to opensearch documentation and if not present, can open an issue in opensearch-project/documentation-website to create the same.

Reply:
Thanks @Poojita-Raj  for catching this, yes you are right. I have temporarily added url of opensearch documentation for cat recovery API, I will update this once I open an issue with opensearch-project/documentation-website